sdk/macro/src/lib.rs

```diff
@@ -213,6 +243,18 @@ pub fn respan(input: TokenStream) -> TokenStream {
     TokenStream::from(to_respan)
 }
 
+#[proc_macro]
+pub fn pubkey(input: TokenStream) -> TokenStream {
+    let id = parse_macro_input!(input as SdkPubkey);
+    TokenStream::from(quote! {#id})
+}
+
+#[proc_macro]
+pub fn program_pubkey(input: TokenStream) -> TokenStream {
```

Comment:
Can you please explain why we need two macros here?

Reply:
Same as for `declare_id` :slightly_smiling_face: 
One for `::solana_program::pubkey::Pubkey`, other for `::solana_sdk::pubkey::Pubkey`.

Reply:
It would be fine to only support `::solana_program::pubkey::Pubkey` here, `::solana_sdk::pubkey::Pubkey` is basically deprecated

Reply:
`::solana_sdk::pubkey::Pubkey` will be removed?
Right now we need two versions because if somebody add `solana-sdk` to dependencies but do not add `solana-program`, then `pubkey!` macro with only `::solana_program::pubkey::Pubkey` will not work.. :/

Reply:
Got it, thanks